src/config.rs

```diff
@@ -345,6 +355,16 @@ fn parse_xpub(s: &str) -> Result<(XyzPubKey, RescanSince)> {
     Ok((xpub, rescan))
 }
 
+fn parse_descriptor(s: &str) -> Result<(String, RescanSince)> {
+    let mut parts = s.splitn(2, ':');
+    let descriptor = String::from(parts.next().unwrap());
+    let rescan = parts
+        .next()
+        .map_or(Ok(RescanSince::Timestamp(0)), parse_rescan)
+        .unwrap();
+    Ok((descriptor, rescan))
+}
```

Comment:
Since we're using RPC to validate / parse descriptors and don't have a client yet, we can't validate the descriptor here